tests/baker_recipes/science.py

```diff
@@ -0,0 +1,22 @@
+from model_bakery.recipe import Recipe, foreign_key
+
+# Recipes for the `science.Degree` objects
+
+degree = Recipe("science.Degree", name="prof.")
+
+
+# Recipes for the `science.Domain` objects
+
+domain = Recipe("science.Domain", name="natural sciences", abbr="SCI")
+
+
+# Recipes for the `science.Discipline` objects
+
+discipline = Recipe(
+    "science.Discipline",
+    name="chemistry",
+    abbr="CH",
+    domain=foreign_key(domain),
+)
+
+__all__ = ["degree", "discipline", "domain"]
```

Comment:
Insert extra line before `__all__`.
```suggestion

__all__ = ["degree", "discipline", "domain"]
```